modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/sidebar_toggler_button/page.jsp

```diff
@@ -16,13 +16,13 @@
 
 <%@ include file="/sidebar_toggler_button/init.jsp" %>
 
-<aui:a cssClass="<%= cssClass %>" href='<%= "#" + sidenavId %>' id="<%= StringUtil.randomId() %>">
```

Comment:
This `id="<%= StringUtil.randomId() %>"` makes me feels goosebumps because IDK what is behind 😂

Reply:
I feel the same but I tested without it and it's working 🤔

Reply:
You are right the id is used to get a real unique selector [here](https://github.com/liferay/liferay-portal/blob/f393dc9ab747829c6345d12b9ce545fff30c67d9/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/side_navigation.es.js#L101-L103), I added back again https://github.com/liferay-lima/liferay-portal/pull/3573/commits/d74790c0bfcae94e2f265dea807e421809876c76.

Thank you!